Sources/ArcGISToolkit/Components/Offline/OfflineManager.swift

```diff
@@ -73,13 +78,41 @@ public class OfflineManager: ObservableObject {
     public static let shared = OfflineManager()
     
     /// The action to perform when a job completes.
-    var jobCompletionAction: ((any JobProtocol) -> Void)?
+    /// - Since 300.0 
+    public var onJobCompletion: ((any JobProtocol) -> Void)?
     
-    /// The job manager used by the offline manager.
-    let jobManager = JobManager(uniqueID: "offlineManager")
+    /// Backing variable for the `jobManager` property.
+    let jobManager: JobManager = JobManager(uniqueID: "offlineManager")
+    
+    /// A Boolean value indicating if the `JobManager` is managing the jobs.
+    var isUsingJobManager: Bool {
```

Comment:
This could be file private.

```suggestion
    fileprivate var isUsingJobManager: Bool {
```